app/uk/gov/hmrc/apiplatformevents/models/jsonFormatters.scala

```diff
@@ -19,10 +19,15 @@ package uk.gov.hmrc.apiplatformevents.models
 
 import play.api.libs.json._
 import uk.gov.hmrc.apiplatformevents.models.common.{Actor, EventId, EventType}
+import uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats
 import uk.gov.hmrc.play.json.Union
 
+import java.time.LocalDateTime
+
 object MongoFormatters {
 
+  implicit val formatLocalDateTimes: Format[LocalDateTime] = MongoJavatimeFormats.localDateTimeFormat
```

Comment:
Can you not just

import uk.gov.hmrc.mongo.play.json.formats.MongoJavatimeFormats.Implicits._

Instead of creating your own here?

Reply:
Not able to do that as the request / json formatters dont use this datetime format... its mongo specific.